docs/githubteacher.md

```diff
@@ -0,0 +1,7 @@
+# GitHub Teacher
+
+**Name:** GitHub Teacher
+
+**Occupation:** Teacher
+ 
+**Location:** San Francisco
```

Comment:
I think we should be more specific.  There are lots of San Franciscos in the :earth_africa: